bonsai-core/src/test/scala/com/stripe/bonsai/GenericTree.scala

```diff
@@ -28,12 +28,10 @@ object GenericTree {
   }
 
   implicit def arbitraryTree[A: Arbitrary]: Arbitrary[GenericTree[A]] =
-    Arbitrary(genTree(arbitrary[A], 5, 12))
+    Arbitrary(genTree(arbitrary[A], 3, 7))
```

Comment:
Were the tests running too slow or failing? Just want to make sure this was intentional, and not a temp thing while working on the tests themselves.